### PR TITLE
Wanted list cartridge

### PR DESCRIPTION
**Pull request by Ertanic**
## About the PR

Adds a pre-installed program to the Security, Captain, and Lawyer PDAs that allows you to view an operational database that contains information on those in processing or with criminal histories. 

In addition to the program itself, a cartridge with a copy of the program has been added to the HOS locker for when a new person joins the Security Department. The cartridge exists as a single copy because the program can be installed on a PDA and the cartridge can be returned.

Therefore, some have suggested adding the cartridge to the thief's targets.  

## Why / Balance

I kept forgetting what reason we were looking for the person, so I thought why not have some sort of reminder. At first I thought of just putting it in the character description, but then I changed my mind and decided to make it as a program in my PDA. And recently I found out that there is something similar in SS13.

## Technical details

It all works directly from the CriminalRecordsSystem, which sends out up-to-date information when records or history changes. There are no changes in CriminalRecordsConsoleSystem, as all that has been done there is to move some code pieces so that the variables are visible when methods are called.

Also, for the icons display I need I had to make a copy of ItemList: StatusList. All it does differently is center the icon and increase its size. I didn't put the icon sizes in a separate property, because the control was created exclusively for this case.

## Media
image

https://github.com/user-attachments/assets/185599c6-5c59-425d-a19d-392d76f54bfd

## Requirements
- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Wanted list program and its cartridge.
- add: The cartridge has been added to the HOS locker.
- add: Added target to thief on wanted list cartridge.

### CODE REVIEW

**Comment by Hanzdegloker** [comment]
Imagine, if the HoS got a special version of the program that allowed them to add and remove people from the criminal records as well as update it, and only the HoS has it.

**Comment by Brandon-Huu** [comment]
Why does it have syndicate logo on it?

File: Resources/Prototypes/Entities/Objects/Devices/pda.yml [hunk 1]
```diff
@@ -795,6 +845,13 @@
     accentVColor: "#447987"
   - type: Icon
     state: pda-ert
+  - type: CartridgeLoader
+    uiKey: enum.PdaUiKey.Key
+    preinstalled:
+    - CrewManifestCartridge
+    - NotekeeperCartridge
+    - NewsReaderCartridge
+    - WantedListCartridge
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Devices/pda.yml:854]: Should this just be using alwayspushinheritance if you need to copy-paste these?

File: Content.Client/CartridgeLoader/UI/StatusList.cs [hunk 2]
```diff
@@ -0,0 +1,674 @@
...
+namespace Content.Client.CartridgeLoader.UI;
+
+public sealed class StatusList : Control, IList<StatusList.StatusItem>
+{
+    private bool _isAtBottom = true;
+
+    /// The size of all the child widgets, in pixels
+    private int _totalContentHeight;
+
+    private VScrollBar _scrollBar;
```
Comment (metalgearsloth) [inline Content.Client/CartridgeLoader/UI/StatusList.cs:20]: Please don't just copy-paste entire classes it's really awful technical debt.

**Comment by UbaserB** [comment]
The CL is too long for a small change like this

**Comment by SlamBamActionman** [comment]
Hello! This PR has gone through a maintainer discussion vote and the result is: Merge, pending code review.

The functionality is great and the new sprite is appropriate. The UI issue with the horizontal scrollbar was noted, however since the PR author expressed uncertainty on how to address it on Discord, we don't feel it's enough of a problem to stop the merge. Once this PR gets merged a Github Issue should be made for the horizontal scroll bar so that it can be touched up later.

Code still requires a review before the PR can be merged, but conceptually this is 👍 and can be merged immediately if the code review passes.

**Comment by DieselMohawk** [comment]
Keep it going man, you got this

**Pull request merged**